docs/skopeo-inspect.1.md

```diff
@@ -4,38 +4,57 @@
 skopeo\-inspect - Return low-level information about _image-name_ in a registry
```

Comment:
(Note to self: I haven’t reviewed this.)

Reply:
Reviewed now. (Would have been a bit easier if the sorting of options were a separate commit.)